pkg/curatedpackages/curatedpackages.go

```diff
@@ -153,3 +154,25 @@ func GetRegistry(uri string) string {
 	}
 	return uri[:lastInd]
 }
+
+func UpgradePackages(
+	ctx context.Context,
+	kubeConfig string,
+	kubectl *executables.Kubectl,
+	source string,
+	targetBundleVersion string,
+) error {
+	b := NewBundleReader(
+		kubeConfig,
+		BundleSource(source),
+		kubectl,
+		nil,
+		nil,
+	)
+	activeController, err := b.GetActiveController(ctx)
```

Comment:
We might want to have some test coverage for this method.

Reply:
The general structure is to have tests in pkg/ but we don't have any tests for cmd/ package. The orchestration is done in cmd/ and the business functionality done in pkg/. I can add the needed tests accordingly since this is just orchestration.